classes/class-wc-connect-nux.php

```diff
@@ -1,5 +1,16 @@
 <?php
 
+if ( ! class_exists( 'Jetpack_Install_Status' ) ) {
+	abstract class Jetpack_Install_Status {
```

Comment:
You don't have to instantiate any defined class, abstract or not. The difference with abstract classes is that they *can not* be instantiated - it causes a fatal error.